scripts/run_backend_tests.py

```diff
@@ -136,6 +137,12 @@
     '--verbose',
     help='optional; if specified, display the output of the tests being run',
     action='store_true')
+_PARSER.add_argument(
+    '--run_on_changed_files',
+    help='optional; if specified, runs the backend tests on the files '
+    'that were changed in the current branch',
```

Comment:
To me --run_on_changed_files sounds like "only from the unstaged files" but the message says it to be for the entire branch, if it's the latter one, should we call it --only_on_changed_files_in_branch (or something around that)?

[`run` is already part of the script name so I'm avoiding it to be part of the flag name)

Reply:
I changed it to --run_on_changed_files_in_branch (kept the run part) and changed it in the frontend tests one, I think with run_ it is better. Though if we shouldn't have it in the flag name should I can change it in the frontend tests one as well but it also has a flag --specs_to_run which has "run" in it as well which I believe we discussed in my proposal to make more obvious that specific things are being "run".